client/my-sites/themes/single-site-wpcom.jsx

```diff
@@ -10,26 +11,41 @@ import CurrentTheme from 'my-sites/themes/current-theme';
 import SidebarNavigation from 'my-sites/sidebar-navigation';
 import ThanksModal from 'my-sites/themes/thanks-modal';
 import { connectOptions } from './theme-options';
-import { FEATURE_ADVANCED_DESIGN } from 'lib/plans/constants';
-import UpgradeNudge from 'my-sites/upgrade-nudge';
+import Banner from 'components/banner';
+import {
+	PLAN_FREE,
+	PLAN_PERSONAL,
+	PLAN_PREMIUM,
+	PLAN_BUSINESS,
+} from 'lib/plans/constants';
+import { getCurrentPlan } from 'state/sites/plans/selectors';
 import QuerySitePlans from 'components/data/query-site-plans';
 import QuerySitePurchases from 'components/data/query-site-purchases';
 import ThemeShowcase from './theme-showcase';
 
-export default connectOptions(
+const ConnectedSingleSiteWpcom = connectOptions(
 	( props ) => {
-		const { siteId, translate } = props;
+		const { siteId, currentPlanSlug, translate } = props;
 
 		return (
 			<div>
 				<SidebarNavigation />
 				<CurrentTheme siteId={ siteId } />
-				<UpgradeNudge
-					title={ translate( 'Get Custom Design with Premium' ) }
-					message={ translate( 'Customize your theme using premium fonts, color palettes, and the CSS editor.' ) }
-					feature={ FEATURE_ADVANCED_DESIGN }
+				{ ( currentPlanSlug === PLAN_FREE || currentPlanSlug === PLAN_PERSONAL ) && <Banner
+					plan={ PLAN_PREMIUM }
+					title={ translate( 'Access all our premium themes with our Premium and Business plans!' ) }
+					description={ translate( 'Get also advanced customization, more space, and video support.' ) }
 					event="themes_custom_design"
-					/>
+				/>
+				}
+				{ ( currentPlanSlug === PLAN_PREMIUM ) && <Banner
+					plan={ PLAN_BUSINESS }
+					title={ translate( 'Why not upgrade to Business?' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Upgrade to Business' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).